src/domain/deliveryLogLines.js

```diff
@@ -0,0 +1,30 @@
+import dbClient from '../utils/dbClient.js'
+
+export default class DeliveryLogLines {
+  static fromDb(deliveryLogLines) {
+    return new DeliveryLogLines(
+      deliveryLogLines.content,
+      deliveryLogLines.logId
+    )
+  }
+
+  static async fromJson(json) {
```

Comment:
Is there a reason this is `async` ?

Reply:
used post.js and user.js as a template, that has this static async from json. Should I remove? Or keep it for consistency?

Reply:
Noted - see above